### DISCUSSION TITLE
Paid Internship Opportunities

### DISCUSSION THREAD

**Discussion by Jay-study-nildana** [Jay-study-nildana/FrontEndForStudents#19] [General]
Students

I always have some or the other work for people with coding skills, if you are interested, just get in touch with me through my socials, links available on my github Profile.

Thanks

Jay.

**Comment by Jay-study-nildana** [comment]
@sukhepadda @Akki-Rawat

Get in touch with me on discord. Add me as friend. I have some paid work for you guys.

Thanks

Discord: Jay_HisManyHobbies#6392

**Reply by Akki-Rawat** [reply]
> @sukhepadda @Akki-Rawat
> 
> Get in touch with me on discord. Add me as friend. I have some paid work for you guys.
> 
> Thanks
> 
> Discord: Jay_HisManyHobbies#6392

I'm already in you discord group.

**Reply by Jay-study-nildana** [reply]
@Akki-Rawat 

Excellent. we have already connected. Thanks.

Jay.

**Comment by Jay-study-nildana** [comment]
Folks

Update. I have an open source project, and hiring interns.  Bangalore based interns only. 

https://medium.com/projectwt/hiring-javascript-and-c-sharp-interns-for-coding-workshop-business-755cb4d617f8

Thanks

Jay (Vijayasimha BR)
Coding Tutor. Career and Life Coach
----
Discord: Jay_HisManyHobbies#6392

**Reply by pranjalgogoi1209** [reply]
Sir, Can I apply for internship now?

**Reply by Jay-study-nildana** [reply]
@pranjalgogoi1209 

We are hiring for Bangalore positions, and for students only. So, if you are based in Bangalore, and if you are currently studying in Bangalore, then you can apply. 

As of now, we don't have any remote openings or positions for graduates. Follow my company, amiay.in, on LinkedIn or Insta where we will update if we have new openings. 

and, if you know any students in Bangalore, you can refer them, of course.

Jay.

**Reply by pranjalgogoi1209** [reply]
@Jay-study-nildana 

Thank you for your response. I am a graduated person. I am searching for my first paid internship. I can relocate to anywhere. In future if you have any opportunity for me, please let me know. I am following you in Youtube, Linkedin, Github and Discord. 

Thank you,
Pranjal Gogoi
Frontend Developer
gogoipranjal2022@gmail.com
https://www.linkedin.com/in/pranjal-gogoi-3aa73b229/

**Reply by Jay-study-nildana** [reply]
@pranjalgogoi1209 

As of now, a small company like mine can only afford student interns :) but, if we open up any opportunities for graduates, I promise to put it on my linked in page and you can apply. 

But, at least 2024, I am sure, we won't be hiring any graduates. We don't have that kind of funding.

Jay.

**Reply by pranjalgogoi1209** [reply]
@Jay-study-nildana 

Okay sir....

**Comment by Akki-Rawat** [comment]
Hey, Jay Sir
how are you? Hope you fine!

Thank you sir for this opportunity, I'm sending my resume to your site and I 'll enjoy to do work with you.

See you soon sir

Thanks
Akki

**Reply by Jay-study-nildana** [reply]
@Akki-Rawat 

I have replied in email. thank you.

Jay
